css-positioning/fixed.css

```diff
@@ -0,0 +1,11 @@
+html {
+  font-size: 22px;
+}
+
+.page-title {
+  font-size: 3rem;
+  text-align: center;
+  position: fixed;
+  top: 100px;
+  right: 500px;
```

Comment:
😱 😱 😱 😱 

Is this to center the text? You can just set the element to be the full width of its containing block, it already has `text-align: center` applied to it.

Reply:
Hehe woops!  Fixed it!
![fixed 2](https://user-images.githubusercontent.com/96809909/151613489-3cc579c5-be05-46cc-b665-1796718e88b9.gif)